packages/orbit-components/src/Loading/types.d.ts

```diff
@@ -12,5 +12,7 @@ export interface Props extends Common.Globals {
   readonly loading?: boolean;
   readonly type?: Type;
   readonly customSize?: number;
+  readonly title?: string;
```

Comment:
I'm not sure about the naming of the prop. Although the value is wrapped in `<title></title>` (in `index.tsx` file), it could be misleading when it's used in another component (ie. in ButtonPrimitive). WDYT?

Reply:
To me, it's ok, if well documented